.github/workflows/cpp-packaging.yml

```diff
@@ -433,7 +433,7 @@ jobs:
           for pkg in artifacts/firebase-cpp-sdk-${{ matrix.sdk_platform }}${{ matrix.suffix }}*-build/*.tgz; do
             # determine the build variant based on the artifact filename
             variant=$(sdk-src/build_scripts/desktop/get_variant.sh "${pkg}")
-            sdk-src/build_scripts/desktop/package.sh -b ${pkg} -o firebase-cpp-sdk-${{ matrix.sdk_platform }}${{ matrix.suffix }}-package -p ${{ matrix.sdk_platform }} -t bin -d ${variant} -P python3
+            sdk-src/build_scripts/desktop/package.sh -b ${pkg} -o firebase-cpp-sdk-${{ matrix.sdk_platform }}${{ matrix.suffix }}-package -p ${{ matrix.sdk_platform }} -t bin -d ${variant} -P python3 -j
```

Comment:
Does this work on machines that might not have "python3" command. I think I have seen cases where if python3 is the only version on box, then we just have "python" command rather than "python3". But if your actions are running on github runners already, I am guessing this is safe.

Reply:
Yeah it runs on the runners just fine already. Is there a python environment variable or similar that I should use here instead?

Reply:
Unfortunately I dont know of any. 
In other python scripts, we had this condition to check for "python3" and if it fails, look for "python".
Another way could be to run "python --version" and query the version but if its running fine for now on the runners, I dont think its a big deal.